test/resource.test.js

```diff
@@ -0,0 +1,373 @@
+            response.body.should.have.property("message");
+            response.body.should.have.property("message").eq("El audio se guardo exitosamente");
+            done();
+        });
+    });
+
+    after((done) => {
+        chai.request(server).delete("/resources")
+        .send({
+            URL: "/audios/6171fde5eb326a2f1850c231.mp3"
+        })
+        .auth(accessToken, { type: 'bearer' })
+        .end( (error, response) => {
+            done();
+        });
+    });
+});
+
+
+describe("PATCH Tests Resource",()=>{
+    before((done) =>{
+        const login = {
+            username: "Karla",
+	        password: "Marst1245#"
+        };
+        chai.request(server).post("/login").send(login)
+        .end( (error, response) => {
+            accessToken = response.body.token;
+            done();
+        });
+    });
+
+    it("PATCH /Resource Bad Request", (done) =>{
+        chai.request(server).patch("/resources")
+        .auth(accessToken, { type: 'bearer' })
+        .end( (error, response) => {
+            response.should.have.status(400);
+            response.body.should.be.a("object");
+            response.body.should.have.property("errors");
+            done();
+        });
+    });
+
+    it("PATCH /Resource Not Found", (done) =>{
+        const url = {
+            URL: "/images/accounts/6164db5823242f430c487fcb.png"
+        }
+        chai.request(server).patch("/resources").send(url)
+        .auth(accessToken, { type: 'bearer' })
+        .end( (error, response) => {
+            response.should.have.status(404);
+            response.body.should.have.property("message");
+            response.body.should.have.property("message").eq("No se encontro registro(s)");
+            done();
+        });
+    });
+
+    it("PATCH /Resource", (done) =>{
+        const url = {
+            URL: "/images/accounts/6164db5823242f430c487fca.png"
+        }
+        chai.request(server).patch("/resources").send(url)
+        .auth(accessToken, { type: 'bearer' })
+        .end( (error, response) => {
+            response.should.have.status(200);
+            done();
+        });
+    });
+});
+
+describe("DELETE Tests Resource",()=>{
+    before((done) =>{
+        const login = {
+            username: "Karla",
+	        password: "Marst1245#"
+        };
+        chai.request(server).post("/login").send(login)
+        .end( (error, response) => {
+            accessToken = response.body.token;
+            done();
+        });
+    });
+
+    it("DELETE /Resource Bad Request", (done) =>{
+        chai.request(server).delete("/resources")
+        .auth(accessToken, { type: 'bearer' })
+        .end( (error, response) => {
+            response.should.have.status(400);
+            response.body.should.be.a("object");
+            response.body.should.have.property("errors");
+            done();
+        });
+    });
+
+    it("DELETE /Resource Bad Request File", (done) =>{
```

Comment:
Codacy has a fix for the issue: [Missing space after =>.](https://app.codacy.com/gh/Martha-Miroslava/APIBiziidiDiidxaza/pullRequest?prid=8292860)
```suggestion
    it("DELETE /Resource Bad Request File", (done) => {
```